client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -73,7 +76,14 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			statusText = this.props.translate( 'scheduled' );
+			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
@Automattic/i18n @akirk thoughts on this? It's the first time I see something like this.

Reply:
Apart from there being a whitespace typo `Scheduled status  with time`, this should really be a comment, not a context.

Reply:
See https://translate.wordpress.com/2016/11/24/context-vs-comment/

Reply:
I'll fix that.

Reply:
fixed

Reply:
Oh, `translate()` doesn't use source code comments but a `comment` key.